This patch fixes a bug with some old versions of apfs.efi, which forgot to calculate handle size.
It should not be necessary with latest 10.14 drivers.

diff --git a/MdePkg/Library/UefiFileHandleLib/UefiFileHandleLib.c b/MdePkg/Library/UefiFileHandleLib/UefiFileHandleLib.c
index 57aad77bc1..2beeef134a 100644
--- a/MdePkg/Library/UefiFileHandleLib/UefiFileHandleLib.c
+++ b/MdePkg/Library/UefiFileHandleLib/UefiFileHandleLib.c
@@ -510,6 +515,8 @@ FileHandleFindFirstFile (
       return (EFI_NOT_FOUND);
     }
     return (Status);
+  } else if ((*Buffer)->Size == 0) {
+    (*Buffer)->Size = BufferSize;
   }
   return (EFI_SUCCESS);
 }
@@ -567,6 +574,8 @@ FileHandleFindNextFile(
   if (BufferSize == 0) {
     FreePool(Buffer);
     *NoFile = TRUE;
+  } else if (Buffer->Size == 0) {
+    Buffer->Size = BufferSize;
   }
 
   return (EFI_SUCCESS);
